app/assets/stylesheets/pages.css.scss

```diff
@@ -104,6 +104,18 @@ $include-html-global-classes: false;
   }
 }
 
+.testimonial {
+  @extend .colored-section;
+  background-image: asset_url("testimonial_background.jpg");
```

Comment:
Gonna ignore this for now, our other use of `asset_url` also uses an underscore.